src/java.base/share/classes/java/lang/ClassValue.java

```diff
@@ -34,6 +40,10 @@
 import static java.lang.ClassValue.ClassValueMap.probeHomeLocation;
 import static java.lang.ClassValue.ClassValueMap.probeBackupLocations;
 
+/*[IF CRIU_SUPPORT]*/
+import openj9.internal.criu.NotCheckpointSafe;
+/*[ENDIF] CRIU_SUPPORT*/
```

Comment:
Please add spaces like in https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/128.

Reply:
Cherry-pick https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/128